src/app/shared/utils/utils.ts

```
@@ -427,6 +427,33 @@ export class Util {
     return this.isEmpty(id) || id === '00000000-0000-0000-0000-000000000000';
   }
 
+  /**
+   * map and rename key in contacts[].address.codeficatorAddressDto -> codeficatorAddress
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)